protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -94,6 +95,21 @@ public function actionEdit()
                     '/admin/group/manage-group-users',
                     'id' => $group->id,
                 ]);
+            } else {
+                if (!empty(Yii::$app->request->post('submitReassignAll'))) {
+                    $this->view->success(Yii::t(
+                        'base',
+                        'Reassigned'
+                    ));
+
+                    $job = new ReassignAllDefaultSpaces();
+                    if (QueueHelper::isQueued($job)) {
+                        $this->view->setStatusMessage('success', Yii::t('AdminModule.information', 'The Space memberships of this group will be reassigned. This process can take up to several minutes.'));
+                    } else {
+                        $job->groupId = $group->id;
+                        Yii::$app->queue->push($job);
```

Comment:
Add the first `Default space reassignment job was queued` here so either it already is queued or its was queued. and instead of  `$this->view->setStatusMessage('success')` use `$this->view->success` or maybe even` $this->view->info`

Reply:
Done!